token/cli/src/main.rs

```diff
@@ -819,7 +764,7 @@ async fn validate_mint(config: &Config<'_>, token: Pubkey) -> Result<Pubkey, Err
 #[allow(clippy::too_many_arguments)]
 async fn command_transfer(
```

Comment:
i commented `comman_transfer` heavily because it was quite a bit of work to follow exactly what all its branches do, if its too much i can remove them tho